app/controllers/application_controller.rb

```diff
@@ -1307,10 +1307,18 @@ def rbac_free_for_custom_button?(task, button_id)
   end
 
   def check_button_rbac
+    task = params[:pressed]
     # buttons ids that share a common feature id
-    common_buttons = %w(rbac_project_add rbac_tenant_add)
-    task = common_buttons.include?(params[:pressed]) ? rbac_common_feature_for_buttons(params[:pressed]) : params[:pressed]
-    # Intentional single = so we can check auth later
+    if params.key? :common_feature
+      case
+      when task.include?("rbac")
```

Comment:
@jzigmund wont this check return "rbac_tenant_add" for any of the buttons that contain string 'rbac' buttons? i only wanted to return "rbac_tenant_add" for rbac_project_add or rbac_tenant_add buttons